AssertionError in version 0.1.9

I'm not sure exactly what triggered this, but I'm getting the following Java exception when running the plugin in PyCharm. Everything keeps running, but I get an error popup, and this stack trace:

`2017-11-02 17:18:25,355 [59338440]  ERROR - aemon.impl.PassExecutorService - (173,173), text: workflow training_data_pipeline {}, language = Language: wdl 
java.lang.AssertionError: (173,173), text: workflow training_data_pipeline {}, language = Language: wdl
	at com.intellij.lang.folding.FoldingDescriptor.<init>(FoldingDescriptor.java:111)
	at com.intellij.lang.folding.FoldingDescriptor.<init>(FoldingDescriptor.java:92)
	at com.intellij.lang.folding.FoldingDescriptor.<init>(FoldingDescriptor.java:77)
	at com.intellij.lang.folding.FoldingDescriptor.<init>(FoldingDescriptor.java:60)
	at winstanley.WdlFoldingBuilder.$anonfun$foldingRegions$2(WdlFoldingBuilder.scala:12)
	at scala.Option.map(Option.scala:146)
	at winstanley.WdlFoldingBuilder.foldingRegions(WdlFoldingBuilder.scala:12)
	at winstanley.WdlFoldingBuilder.$anonfun$foldingRegions$1(WdlFoldingBuilder.scala:12)
	at winstanley.WdlFoldingBuilder.$anonfun$foldingRegions$1$adapted(WdlFoldingBuilder.scala:12)
	at scala.collection.TraversableLike.$anonfun$flatMap$1(TraversableLike.scala:241)
	at scala.collection.IndexedSeqOptimized.foreach(IndexedSeqOptimized.scala:32)
	at scala.collection.IndexedSeqOptimized.foreach$(IndexedSeqOptimized.scala:29)
	at scala.collection.mutable.ArrayOps$ofRef.foreach(ArrayOps.scala:193)
	at scala.collection.TraversableLike.flatMap(TraversableLike.scala:241)
	at scala.collection.TraversableLike.flatMap$(TraversableLike.scala:238)
	at scala.collection.mutable.ArrayOps$ofRef.flatMap(ArrayOps.scala:193)
	at winstanley.WdlFoldingBuilder.foldingRegions(WdlFoldingBuilder.scala:12)
	at winstanley.WdlFoldingBuilder.buildFoldRegions(WdlFoldingBuilder.scala:15)
	at com.intellij.lang.folding.LanguageFolding.buildFoldingDescriptors(LanguageFolding.java:83)
	at com.intellij.codeInsight.folding.impl.FoldingUpdate.a(FoldingUpdate.java:241)
	at com.intellij.codeInsight.folding.impl.FoldingUpdate.getFoldingsFor(FoldingUpdate.java:227)
	at com.intellij.codeInsight.folding.impl.FoldingUpdate.a(FoldingUpdate.java:112)
	at com.intellij.codeInsight.folding.impl.FoldingUpdate.a(FoldingUpdate.java:101)
	at com.intellij.psi.impl.PsiParameterizedCachedValue.doCompute(PsiParameterizedCachedValue.java:45)
	at com.intellij.util.CachedValueBase.getValueWithLock(CachedValueBase.java:222)
	at com.intellij.psi.impl.PsiCachedValue.getValueWithLock(PsiCachedValue.java:60)
	at com.intellij.psi.impl.PsiParameterizedCachedValue.getValue(PsiParameterizedCachedValue.java:35)
	at com.intellij.psi.util.CachedValuesManager.getParameterizedCachedValue(CachedValuesManager.java:96)
	at com.intellij.codeInsight.folding.impl.FoldingUpdate.updateFoldRegions(FoldingUpdate.java:97)
	at com.intellij.codeInsight.folding.impl.CodeFoldingManagerImpl.a(CodeFoldingManagerImpl.java:326)
	at com.intellij.codeInsight.folding.impl.CodeFoldingManagerImpl.updateFoldRegionsAsync(CodeFoldingManagerImpl.java:312)
	at com.intellij.codeInsight.daemon.impl.CodeFoldingPass.doCollectInformation(CodeFoldingPass.java:42)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:70)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.a(PassExecutorService.java:438)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1148)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.c(PassExecutorService.java:431)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:548)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:493)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:94)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.d(PassExecutorService.java:430)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.b(PassExecutorService.java:406)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:142)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:237)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:404)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:165)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
2017-11-02 17:18:25,356 [59338441]  ERROR - aemon.impl.PassExecutorService - PyCharm 2017.2.4  Build #PY-172.4343.24 
2017-11-02 17:18:25,356 [59338441]  ERROR - aemon.impl.PassExecutorService - JDK: 1.8.0_152-release 
2017-11-02 17:18:25,356 [59338441]  ERROR - aemon.impl.PassExecutorService - VM: OpenJDK 64-Bit Server VM 
2017-11-02 17:18:25,356 [59338441]  ERROR - aemon.impl.PassExecutorService - Vendor: JetBrains s.r.o 
2017-11-02 17:18:25,356 [59338441]  ERROR - aemon.impl.PassExecutorService - OS: Mac OS X 
2017-11-02 17:18:25,356 [59338441]  ERROR - aemon.impl.PassExecutorService - Last Action: EditorBackSpace 
`

--- a/src/winstanley/structure/WdlImplicits.scala
+++ b/src/winstanley/structure/WdlImplicits.scala
@@ -21,7 +21,7 @@ object WdlImplicits {
         } yield new TextRange(lbrace.getTextRange.getStartOffset + 1, rbrace.getTextRange.getEndOffset - 1)
       }
 
-      psiElement match {
+      val potentialRange = psiElement match {
         case _: WdlTaskBlock | _: WdlWorkflowBlock | _: WdlTaskOutputs | _: WdlWfOutputs | _: WdlCallBlock | _: WdlScatterBlock | _: WdlIfStmt =>
           interBraceContentRange(psiElement, WdlTypes.LBRACE, WdlTypes.RBRACE)
         case wcb: WdlCommandBlock =>
@@ -30,6 +30,8 @@ object WdlImplicits {
           mapContainingContentRange(psiElement)
         case _ => None
       }
+
+      potentialRange collect { case range if range.getLength > 0 => range }
     }
 
     def findContainingScatter: Option[WdlScatterBlock] = {

